src/App.tsx

```diff
@@ -1,24 +1,164 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { addTodo, getTodos, deleteTodo } from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+const USER_ID = 7011;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
+  const [selectedStatus, setSelectedStatus] = useState<Status>(Status.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isDisabledInput, setIsDisabledInput] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [deletingTodoId, setDeletingTodoId] = useState(0);
+  const [isDeletingCompleted, setIsDeletingCompleted] = useState(false);
+
+  const isActiveTodos = todos.filter(todo => !todo.completed);
+
+  function clearErrorMessage() {
+    setTimeout(() => {
+      setErrorMessage('');
+    }, 3000);
+  }
```

Comment:
small tip: you can call this function in `useEffect` and don't add each time after setting the error
```suggestion
  useEffect(() => {
    const clearErrorMessage = () => {
      setTimeout(() => {
        setErrorMessage('');
      }, 3000);
    }
  
    if (errorMessage && errorMessage.length > 0) {
      clearErrorMessage();
    }
  }, [errorMessage]);
```